js/controllers/form/DOIPubIdExportFormHandler.js

```diff
@@ -25,10 +25,34 @@
 			function($formElement, options) {
 		this.parent($formElement, options);
 
-		$('#exportSubmissionXmlForm a[id*="-button-"]').click(this.callbackWrapper(
-				this.submitAction_));
-		$('#exportIssueXmlForm a[id*="-button-"]').click(this.callbackWrapper(
-				this.submitAction_));
+		this.submissionsActions_ = [];
+		this.issuesActions_ = [];
+		this.actionRegExp_ = [];
+
+		var i;
+		if (options.submissionsActions) {
+			for (i = 0; i < options.submissionsActions.length; i++) {
+				/** @type {{_id:string}}*/
+				this.optionsSubmissionsAction = options.submissionsActions[i];
+				this.submissionsActions_.push(this.optionsSubmissionsAction._id);
+				$('#exportSubmissionXmlForm a[id^="' + this.optionsSubmissionsAction._id +
+					'-button-"]').click(this.callbackWrapper(this.submitAction_));
+			}
+			this.actionRegExp_.exportSubmissionXmlForm =
+					'(' + this.submissionsActions_.join('|') + ')';
+		}
+
+		if (options.issuesActions) {
+			for (i = 0; i < options.issuesActions.length; i++) {
+				/** @type {{_id:string}}*/
+				this.optionsIssuessAction = options.issuesActions[i];
+				this.issuesActions_.push(this.optionsIssuessAction._id);
```

Comment:
Ah, sorry @asmecher, I solved it in my recent PR for medra, s. https://github.com/pkp/ojs/pull/949/commits/8e1a79ad2b46f132ee19b25928326b29282eca95#diff-7 I will close this PR and add the changes according to your comments heere there, OK?